src/components/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ButtonProps } from './types';
+import styles from './Button.css';
```

Comment:
❗️
В одном из вышеидущих комментов я предложил добавить плагин для систематизации импортов. Например, очень рекомендую отделять пустой строкой внешние (зависимости проекта) и внутренние (всё, что есть в этой же репе) импорты, а также отделять пустой стили, как ниже
```suggestion
import classNames from 'classnames';

import { ButtonProps } from './types';

import styles from './Button.css';
```

Reply:
Исправлено 423653d1abd5a328e1464dd347e907c871fe3c85